src/nix/devcontainer-feature.json

```diff
@@ -7,7 +7,10 @@
     "options": {
         "version": {
```

Comment:
Can we bump the minor version? Thanks! 

https://github.com/devcontainers/features/blob/260aeae1c8b1aa0a6da18aa96b924b92d6de7743/src/nix/devcontainer-feature.json#L3

Reply:
As you wish 😺